Cartfile

```diff
@@ -1,7 +1,7 @@
 binary "https://www.mapbox.com/ios-sdk/MapboxAccounts.json" ~> 2.2
 binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" ~> 5.6
-binary "https://www.mapbox.com/ios-sdk/MapboxNavigationNative.json" ~> 6.2.1
-github "mapbox/mapbox-directions-swift" ~> 0.31
+binary "https://www.mapbox.com/ios-sdk/MapboxNavigationNative.json" ~> 9.0.3
```

Comment:
Pinned to v9.0.4 specifically in 5cae87dc409568826a1dc20b1734f66fccc39a41 and aa149ed5cb3fc00f0b2888c44f9ed32067535d59, because v9.2.1 is available and not backwards-compatible. Compatibility with v9.2.1 would require a bit of work, but we’re doing that anyways as part of upgrading to v13._x_.

/cc @mapbox/navnative